FeedStoreChallenge/Core Data Store/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Hitender Kumar on 22/05/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+internal class ManagedFeedImage: NSManagedObject {
```

Comment:
`ManagedFeedImage` is not supposed to be inherited thus it should be `final`

Please make it `final`.

```suggestion
final class ManagedFeedImage: NSManagedObject {
```

Reply:
done